tools/cldr-code/src/main/java/org/unicode/cldr/util/ZoneParser.java

```diff
@@ -672,6 +672,7 @@ private List<String> getData(String s) {
             { "Asia/Ho_Chi_Minh", "Asia/Saigon" },
             { "Asia/Yangon", "Asia/Rangoon" },
             { "Asia/Kathmandu", "Asia/Katmandu" },
+            { "Europe/Kyiv", "Europe/Kiev" },
```

Comment:
@pedberg-icu I figured out we need new/old zone ID mapping in this code. This code is used for collecting zone IDs from tzdata files.

Reply:
Ah, thanks for figuring this out. Sorry, I have been in meetings all afternoon...